lib/src/main/java/com/basistheory/android/service/CardBrandEnricher.kt

```diff
@@ -7,14 +7,21 @@ internal class CardBrandEnricher {
     object CardMasks {
         const val MASK_4_8_12GAPS_19LENGTH = "#### #### #### #######"
         const val MASK_4_8_12GAPS_16LENGTH = "#### #### #### ####"
+        const val MASK_4_10GAPS_15LENGTH = "#### ###### #####"
+        const val MASK_4_10GAPS_19LENGTH = "#### ###### #########"
+    }
+
+    object CvcMasks {
+        const val MASK_3LENGTH = "###"
```

Comment:
I used THREE_DIGIT instead of the same name as card number masks, which i did in that way cuz of the gaps + length

Reply:
👍 changed to your names